test/integration/platforms/test_slurmplatform.py

```diff
@@ -359,4 +380,4 @@ def test_run_all_wrappers_workflow_slurm(autosubmit_exp: AutosubmitExperimentFix
     }
 
     exp.autosubmit._check_ownership_and_set_last_command(exp.as_conf, exp.expid, 'run')
-    assert 0 == exp.autosubmit.run_experiment(_EXPID)
+    assert 0 == exp.autosubmit.run_experiment(exp.expid)
```

Comment:
@VindeeR , ignore the rest here for now. This file contains some changes that I expect will make the test_slurm stop failing in this branch :+1: You may copy to one of your other branches to confirm it fixes the intermittent failures.